### PR TITLE
feat: version field

### LINKED ISSUE #8: Add support for optional parameter "version"
Desired usage:
```
uses: Informasjonsforvaltning/upload-files-to-static-rdf-server-action@v3.2.0
with:
  ontology-type: "vocabulary"
  ontology: "hello-world"
  version: "v1.1"
  files: |
    testfiles/hello-world.ttl text/turtle en
    testfiles/hello-world-nb.ttl text/turtle nb
  host: "https://example.com"
  api-key: ${{ secrets.API_KEY }}
```

If given, value of `version` should be last part of url:
```
url="${host}/${ontology_type}/${ontology}/${version}"
```
If `version` is undefined, url should be as before:
```
url="${host}/${ontology_type}/${ontology}"
```

### PR DESCRIPTION
**Pull request by tenstad**
Closes https://github.com/Informasjonsforvaltning/upload-files-to-static-rdf-server-action/issues/8

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -8,6 +8,9 @@ inputs:
   ontology:
     description: The name of the ontology, e.g. vocabulary.
     required: true
+  version:
+    description: The version ..?
```
Comment (tenstad) [inline action.yml]: @stigbd suggestion pls

Comment (stigbd) [reply action.yml]: Suggestion:
The version of the specification. Should not be used for latest version

**Review by stigbd — DISMISSED** [review]
Nice!

File: action.yml (hunk 1 above)
Comment (stigbd) [reply action.yml]:
```suggestion
    description: The version of the specification. Should not be used for latest version.
```

**Review by stigbd — APPROVED** [review]

**Pull request merged**